presto-hive/src/main/java/io/prestosql/plugin/hive/HiveSplit.java

```diff
@@ -50,7 +50,7 @@
     private final TableToPartitionMapping tableToPartitionMapping;
     private final Optional<BucketConversion> bucketConversion;
     private final boolean s3SelectPushdownEnabled;
-    private final Optional<DeleteDeltaLocations> deleteDeltaLocations;
+    private final Optional<AcidInfo> acidInfo;
```

Comment:
Note: Would be nice to separate out a commit introducing `AcidInfo` class, separately from other changes.
You can do this on your own, or reuse first commit from https://github.com/prestosql/presto/pull/4049.

Reply:
Yeah, I will create a separate commit introducing AcidInfo class.

Reply:
it seems @wendigo took care of this in #4049. I think he picked the `AcidInfo` name to avoid causing conflicts with your PR, so you should just be able to rebase.

Reply:
I have rebased PR on the master branch, thank you.